docs/internals/contributing/bugs-and-features.txt

```diff
@@ -126,45 +126,29 @@ benchmarks.
 How we make decisions
 =====================
 
-Whenever possible, we strive for a rough consensus. To that end, we'll often
-have informal votes on the Django Forum about a feature. In these votes we
-follow the voting style invented by Apache and used on Python itself, where
-votes are given as +1, +0, -0, or -1.
-Roughly translated, these votes mean:
-
-* +1: "I love the idea and I'm strongly committed to it."
-
-* +0: "Sounds OK to me."
-
-* -0: "I'm not thrilled, but I won't stand in the way."
-
-* -1: "I strongly disagree and would be very unhappy to see the idea turn
-  into reality."
```

Comment:
I wonder if we lose something by removing this
It's certainly existed in other (historic) discussions and might still be used when someone summaries their opinion